common/features/config/networks/static/reducer.ts

```diff
@@ -999,6 +1000,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       initial: 4.77
     },
     shouldEstimateGasPrice: false
+  },
+  AUX: {
+    id: 'AUX',
+    name: 'Auxilium',
+    unit: 'AUX',
+    chainId: 28945486,
+    isCustom: false,
+    color: '#85dc35',
+    blockExplorer: makeExplorer({
+      name: 'Auxilium Explore',
+      origin: 'https://explore.auxilium.global/'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: AUX_DEFAULT,
```

Comment:
Please do not advertise Trezor support without actually supporting the Trezor hardware wallet.  You should remove this line for now - you may add it back in a separate pull request later on if Trezor releases support for your chain.